frontend/src/libs/components/profile-settings-sidebar/profile-settings-sidebar.tsx

```diff
@@ -0,0 +1,69 @@
+import { Icon } from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+import { useAppSelector, useCallback, useState } from '#libs/hooks/hooks.js';
+import { type IconName } from '#libs/types/types.js';
+import { type UserAuthResponseDto } from '#packages/users/users.js';
+
+import { ProfileSettingsButton } from './profile-settings-button/profile-settings-button.js';
+import styles from './styles.module.scss';
+
+const ProfileSettingsSidebar: React.FC = () => {
+  const userName = useAppSelector((state) => {
+    const user = state.auth.authenticatedUser as UserAuthResponseDto;
+
+    return user.fullName;
+  });
```

Comment:
```suggestion
  const { userName } = useAppSelector((state) => {
    return => {
       userName:  (state.auth.authenticatedUser as UserAuthResponseDto).fullName;
    }
  });
```
We just get used to this approach

Reply:
may be it`s better to do smth like this:
  const fullName = useAppSelector(({ auth }) => {
    return (auth.authenticatedUser as UserAuthResponseDto).fullName;
  });

Reply:
May be also an option, we just used object destruction in every app selector I saw